data-prepper-core/src/main/java/com/amazon/dataprepper/parser/config/DataPrepperAppConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.parser.config;
+
+import com.amazon.dataprepper.parser.model.DataPrepperConfiguration;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+import java.io.File;
+import java.io.IOException;
+
+@Configuration
+public class DataPrepperAppConfiguration {
+    private static final Logger LOG = LoggerFactory.getLogger(DataPrepperAppConfiguration.class);
+    private static final String POSITIONAL_COMMAND_LINE_ARGUMENTS = "nonOptionArgs";
```

Comment:
This line was somewhat confusing, but clear once I looked this string up on Google. Spring appears to provide this constant, so it would probably be clearer to just use it:

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/core/env/CommandLinePropertySource.html#DEFAULT_NON_OPTION_ARGS_PROPERTY_NAME

Reply:
Good find!